package.json

```diff
@@ -34,6 +34,7 @@
   "devDependencies": {
     "babel-cli": "^6.10.1",
     "babel-eslint": "^6.0.5",
+    "babel-plugin-transform-regenerator": "^6.21.0",
```

Comment:
I don't know... maybe you need to clean your npm cache. I think it's `npm cache clean`. </shot-in-the-dark>